src/conteiners/MovieListContainer.js

```diff
@@ -0,0 +1,20 @@
+import { connect } from 'react-redux';
+import MovieList from '../components/MovieList/MovieList';
+import getFilms from '../modules/actions/getFilms';
+import { getListFilms, getGenre } from '../modules/selectors/index';
+
+const mapStateToPops = (state) => {
+  return {
+    films: getListFilms(state),
+    genre: getGenre(state),
+  }
+}
+
+
+const MovieListContainer = connect(mapStateToPops, dispatch => ({
```

Comment:
in case using redux hooks we don't need to use container pattern

Reply:
the same here